index.html

```diff
@@ -990,32 +656,24 @@ <h2>
     Examples
   </h2>
   <pre class="example js" title="Adjusting the number of video feeds based on CPU usage">
-    const observer = new ComputePressureObserver(computePressureCallback);
+    let timerId = -1;
+    function pressureChange(entries) {
+      clearTimeout(timerId);
 
-    observer.observe("cpu", {
-      cpuUtilizationThresholds: [0.75, 0.9, 0.5],
-      cpuSpeedThresholds: [0.5],
-    });
+      const lastEntryArray = [entries.at(entries.length - 1)];
+      timerId = setTimeout(pressureChange.bind(this, lastentryArray), 30_000);
```

Comment:
Can you add a comment why you're manually calling `pressureChange` in 30s?

Reply:
I totally missed that part. Added now

Reply:
I don't see a comment.

Reply:
It's not a comment, its above:

  <p>
    In this example we want to lower the number of concurrent video streams if pressure
    becomes critical. For the same of simplicity we only consider this one state.
  </p>
  <p>
    As lowering the amount of streams might not result in exiting the critical state,
    or at least not immediately, we use a strategy where we lower one stream at the time
    every 30 seconds while still in the critical state.
  </p>
  <p>
    We accomplish this by making sure the callback is called at least once every 30 seconds,
    or when the state actually changes. When the state changes we reset the interval timer.
  </p>

Reply:
It wasn't clear when reading this. Please add some comments that explains what you're actually doing.

Reply:
OK let me try